packages/select/src/Select.tsx

```diff
@@ -38,8 +54,11 @@ const errorTextStyle = ({
   darkMode: boolean;
   sizeSet: SizeSet;
 }) => css`
-  font-family: ${darkMode ? fontFamilies.legacy : fontFamilies.default};
-  color: ${darkMode ? '#F97216' : palette.red.base};
+  font-family: ${fontFamilies.default};
+  color: ${darkMode
+    ? '#FF6960'
+    : palette.red
+        .base}; // TODO:palette.red.light1 is different in figma(#FF6960) which does not match react(#EF5752)
```

Comment:
Same thing here. Unless we add it to the ticket to update these tokens, my feeling is that we should leave this as the token, and have it magically get fixed

Reply:
true! i'm going to be working on that ticket this week anyways so makes sense